AutoRunCustomToolShared/VSPackage.resx

```diff
@@ -134,7 +134,4 @@
   <data name="112" xml:space="preserve">
     <value>Automatically runs the custom tool on another file when a file is modified.</value>
   </data>
-  <data name="400" type="System.Resources.ResXFileRef, System.Windows.Forms">
-    <value>Resources\Package.ico;System.Drawing.Icon, System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a</value>
-  </data>
```

Comment:
I don't think this should be removed. This resource is [referenced from the package](https://github.com/thomaslevesque/AutoRunCustomTool/blob/48fc68798dcd7ee4fbaea07d9bab8817c0a7cfee/AutoRunCustomTool/AutoRunCustomToolPackage.cs#L19).
There might be some tweaking to do in the project or resx file to include the file from Shared\Resources. If it turns out to be difficult, just leave the duplicate icon.

Reply:
Couldn't find an easy way to include this resource without duplicating it in the shared folder sorry.
I've added the duplicate icon back.